### DISCUSSION TITLE
Get Started with EcoEye–Embedded Vision Camera | Seeed Studio Wiki

### DISCUSSION THREAD

**Discussion by giscus** [Seeed-Studio/wiki-documents#1010] [Q&A]
# Get Started with EcoEye–Embedded Vision Camera | Seeed Studio Wiki

Get_Started_with_EcoEye–Embedded_Vision_Camera

https://wiki.seeedstudio.com/Get_Started_with_EcoEye_Embedded_Vision_Camera/

**Comment by ELBE223** [comment]
Hello everyone,

I'm working on a project using the EcoEye and following the steps in this guide here but I keep running into an issue. Every time I try to execute my script, I receive the following error:

Copy code
Traceback (most recent call last): File "<stdin>", line 206, in <module> File "<string>", line 95, in <module> SystemExit: ERROR: RTC on Solarboard not connected ! OpenMV v4.5.3; MicroPython v1.22-omv.r17; OPENMV4P-STM32H743

I'm not sure how to proceed and would appreciate any advice. Has anyone encountered this error before? 
Thank you in advance for your help!
